### PR TITLE
Feature fa qs custom training

### PR DESCRIPTION
**Pull request by dieguischa**

### CODE REVIEW

**Review by DanielOlarte — CHANGES_REQUESTED** [review]
Let's use the correct nesting for the new routes (api/voicebots/:voicebot_id) because in some of the new endpoints for intents you have the new format and others have the older format

**Review by DanielOlarte — APPROVED** [review]

File: src/components/Utils/ItemRow.jsx [hunk 1]
```diff
@@ -14,26 +14,35 @@ const ItemRow = (props) => {
     setItem(event.target.value);
   }
 
+  const getHelperText = (item) => {
+    if (props.helperText) {
+      return item.length + "/" + props.maxLength;
```
Comment (juan-voiq) [inline src/components/Utils/ItemRow.jsx:19]: Lets use interpolation please, it is easier to read

File: src/components/Voicebots/EditVoicebot/Setup/Train/TrainingMaterial/Examples.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      }
+    );
+  }
+
+  const onExampleBlur = (event) => {
+    event.persist();
+
+    IntentExamplesService.update(
+      selectedUtterance,
+      event.target.id.split('-')[1],
```
Comment (juan-voiq) [inline src/components/Voicebots/EditVoicebot/Setup/Train/TrainingMaterial/Examples.jsx:49]: This splitting its a little obscure, can we create a helper function with a comment that explains why it is the second item of the splitted string?

**Review by juan-voiq — CHANGES_REQUESTED** [review]
Couple of small changes

**Pull request merged**